src/components/AddActor.jsx

```diff
@@ -3,42 +3,43 @@ import React, {useState} from 'react';
 const AddActor = ({actors, setActors}) => {
   const [addActorError, setAddActorError] = useState('');
 
-const newActorHandler = (e) => {
-  e.preventDefault()
-  const newActor = e.target.elements.actorName.value;
-  // Add the new actor to the database, returning the new actor object
-  fetch('/actors', {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/json',
-    },
-    body: JSON.stringify({name: newActor}),
-  })
-    .then((response) => {
-      if (response.ok) {
-        return response.json();
-      } else {
-        throw new Error('Error when adding the actor');
-      }
+  const newActorHandler = (e) => {
+    e.preventDefault()
+    const newActor = e.target.elements.actorName.value;
+    // Add the new actor to the database, returning the new actor object
+    fetch('/actors', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ name: newActor }),
     })
-    .then((actor) => {
-      // add the actor to the actor list and reset values
-      setActors([...actors, actor])
-      setAddActorError('');
-      e.target.reset();
-    })
-    .catch((error) => {
-      setAddActorError(error.message);
-    });
-};
+      .then((response) => {
+        if (response.ok) {
+          return response.json();
+        } else {
+          throw new Error('Error when adding the actor');
+        }
+      })
+      .then((actor) => {
+        // add the actor to the actor list and reset values
+        setActors([...actors, actor])
+        setAddActorError('');
+        e.target.reset();
+      })
+      .catch((error) => {
+        setAddActorError(error.message);
+      });
+  };
 
   return (
-      <form onSubmit={newActorHandler}>
-          <input name="actorName" placeholder="enter actor's name" />
-          <div style={{ color: 'red' }}>{addActorError}</div>
-          <button type="submit">Add actor</button>
-      </form>
+    <form onSubmit={newActorHandler}>
+      <h3>Add a new actor</h3>
+      <input name="actorName" placeholder="enter actor's name" />
+      <div className='error'>{addActorError}</div>
+      <button type="submit">Add actor</button>
+    </form>
   )
 }
 
-export default AddActor
\ No newline at end of file
+export default AddActor;
```

Comment:
Github is complaining about some of these files not ending in a newline (the red - in a circle). It's good practice to leave a newline at the end of a file since that improves compatibility with many command line text tools.

Reply:
Good to know, I saw that new line error and was confused why that was an issue.